### PR TITLE
CLI for creating config.yaml

### PR DESCRIPTION
**Pull request by nkoenig**
# 🎉 New feature

## Summary

This simplifies the process of creating a config.yaml file by supporting the `gz fuel configure` CLI. This new CLI walks a user through the process of creating a config.yaml file that will live in `~/.gz/fuel/`. 

The primary benefit is that it makes it easier to store and then use private access tokens.

## Test it

1. Run the full guided process with:
```
gz fuel configure
```
2. Accept the defaults, and create a standard config file.
```
gz fuel configure --defaults
```
3. Accept the defaults, and output the result to the console.
```
gz fuel configure --defaults --console
```

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [x] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,25 @@ Downloading model:
 Download succeeded.
 ```
 
+**Private access tokens**
+Private models and worlds can be downloaded using access tokens.
+Access tokens are generated on `app.gazebosim.org`. After logging in,
+go to `Settings->Access Tokens`.
+
+An access token can be used with CLI commands via the `--header` option:
+```
```
Comment (ahcorde) [inline README.md:67]:
```suggestion
```bash
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,25 @@ Downloading model:
...
 
+**Private access tokens**
+Private models and worlds can be downloaded using access tokens.
+Access tokens are generated on `app.gazebosim.org`. After logging in,
+go to `Settings->Access Tokens`.
+
+An access token can be used with CLI commands via the `--header` option:
+```
+$ gz fuel download -u https://fuel.gazebosim.org/1.0/openrobotics/models/ambulance --header 'Private-Token: <access_token>'
+
```
Comment (ahcorde) [inline README.md:69]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -58,6 +58,25 @@ Downloading model:
...
+```
+
+Or, an access token can be stored in a `~/.gz/fuel/config.yaml` file. The token is then
+automatically used by the command line tool and API calls. Use the `configure` helper
+tool create your `~/.gz/fuel/config.yaml` file.
+
+```
```
Comment (ahcorde) [inline README.md:76]:
```suggestion
```bash
```

File: tutorials/02_configuration.md [hunk 4]
```diff
@@ -35,6 +35,17 @@ The `cache` section captures options related with the local storage of the
...
 downloaded. If not used, all assets are stored under `$HOME/.gz/fuel`.
 
+## Guided Configuration
+
+The `gz fuel configure` CLI will walk you through the process of creating a 
+`~/.gz/fuel/config.yaml` file. Just run the following command, and answer
+the prompts. Note that this command will replace your existing `~/.gz/fuel/config.yaml`
+if you choose to save on the last step.
+
+```
```
Comment (ahcorde) [inline tutorials/02_configuration.md:45]:
```suggestion
```bash
```

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: tutorials/02_configuration.md (hunk 4 above)
Comment (nkoenig) [reply tutorials/02_configuration.md]: [98719eb](https://github.com/gazebosim/gz-fuel-tools/pull/413/commits/98719eb12f242df49e9cfa2998ce19673db3ad05)

File: README.md (hunk 3 above)
Comment (nkoenig) [reply README.md]: [98719eb](https://github.com/gazebosim/gz-fuel-tools/pull/413/commits/98719eb12f242df49e9cfa2998ce19673db3ad05)

File: README.md (hunk 1 above)
Comment (nkoenig) [reply README.md]: [98719eb](https://github.com/gazebosim/gz-fuel-tools/pull/413/commits/98719eb12f242df49e9cfa2998ce19673db3ad05)

**Review by ahcorde — APPROVED** [review]

**Pull request merged**